chapter1/Voting.sol

```diff
@@ -19,12 +19,12 @@ contract Voting {
   }
 
   function totalVotesFor(bytes32 candidate) returns (uint8) {
-    if (validCandidate(candidate) == false) throw;
```

Comment:
Per this: http://solidity.readthedocs.io/en/develop/control-structures.html#error-handling-assert-require-revert-and-exceptions, it is better to use `require` method instead of `revert`. Do you agree?